docs/_tooling/extensions/modularity/filter_overwrite.py

```diff
@@ -0,0 +1,82 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+from sphinx.application import Sphinx
+from sphinx_needs.views import NeedsView
+from sphinx_needs.data import NeedsInfoType, NeedsFilteredBaseType
+from sphinx_needs.config import NeedsSphinxConfig
+from sphinx_needs.views import NeedsAndPartsListView
+from docutils import nodes
+from sphinx_needs.debug import measure_time
+
+
+#                    ╭──────────────────────────────────────────────────────────────────────────────╮
+#                    │             Wrapping filter funcs to enable 'hide' to be respected           │
+#                    ╰──────────────────────────────────────────────────────────────────────────────╯
+
```

Comment:
I'm sure this was meant to look nice or fancy... but it doesn't ;-)

![image](https://github.com/user-attachments/assets/5a535056-21e9-4ab7-ba19-ae1c1fdc58c3)